helpdesk_mgmt/tests/test_helpdesk_portal.py

```diff
@@ -47,30 +46,32 @@ def setUp(self):
     def get_new_tickets(self, user):
         return self.env["helpdesk.ticket"].with_user(user).search([])
 
-    def _create_ticket(self, partner, ticket_title):
+    def _create_ticket(self, partner, ticket_title, **values):
         """Create a ticket submitted by the specified partner."""
-        return self.env["helpdesk.ticket"].create(
-            {
-                "name": ticket_title,
-                "description": "test-description",
-                "partner_id": partner.id,
-                "partner_email": partner.email,
-                "partner_name": partner.name,
-            }
-        )
+        data = {
+            "name": ticket_title,
+            "description": "test-description",
+            "partner_id": partner.id,
+            "partner_email": partner.email,
+            "partner_name": partner.name,
+        }
+        data.update(**values)
+        return self.env["helpdesk.ticket"].create(data)
 
-    def _submit_ticket(self):
-        resp = self.url_open(
-            "/submitted/ticket",
-            data={
-                "category": self.env.ref("helpdesk_mgmt.helpdesk_category_1").id,
-                "csrf_token": http.WebRequest.csrf_token(self),
-                "subject": self.new_ticket_title,
-                "description": "\n".join(self.new_ticket_desc_lines),
-            },
-        )
+    def _submit_ticket(self, **values):
+        data = {
+            "category": self.env.ref("helpdesk_mgmt.helpdesk_category_1").id,
+            "csrf_token": http.WebRequest.csrf_token(self),
+            "subject": self.new_ticket_title,
+            "description": "\n".join(self.new_ticket_desc_lines),
+        }
+        data.update(**values)
+        resp = self.url_open("/submitted/ticket", data=data)
         self.assertEqual(resp.status_code, 200)
 
+
+@odoo.tests.tagged("post_install", "-at_install")
```

Comment:
Why this?

Reply:
Because the original TestHelpdeskPortal in `helpdesk_mgmt` was already tagged with this, I assumed it was essential for any HttpCase.

Reply:
OK, I see, but why not keeping it in the base class? Isn't working that way?

Reply:
Sorry, I missed the question.
If I remember, I made a quick code audit (odoo/oca) to see how base/common test classes were handled in such cases.
Each time, the base class don't have any decorator, only child classes have.

Reply:
Well, but if it's working, it's better to not increase the diff. Here there's no guideline about this.

Reply:
Ok I made a test and the decorator is not taken into account if set on the base class.

Reply:
OK then. Good to know. Weird, but if happening, indeed it should be changed.